Tests/DatadogIntegrationTests/Scenarios/RUM/RUMResourcesScenarioTests.swift

```diff
@@ -113,13 +113,17 @@ class RUMResourcesScenarioTests: IntegrationTests, RUMCommonAsserts {
 
         // Get RUM Sessions with expected number of View visits and Resources
         let rumRequests = try rumServerSession.pullRecordedRequests(timeout: dataDeliveryTimeout) { requests in
-            try RUMSessionMatcher.singleSession(from: requests)?.viewVisits.count == 2
+            let session = try RUMSessionMatcher.singleSession(from: requests)
+            let hasAllViews = session?.viewVisits.count == 2
+            let hasAllResources = session?.resourceEventMatchers.count == 4
+            let hasAllErrors = session?.errorEventMatchers.count == 1
+            return hasAllViews && hasAllResources && hasAllErrors
         }
 
         assertRUM(requests: rumRequests)
 
         let session = try XCTUnwrap(try RUMSessionMatcher.singleSession(from: rumRequests))
-        XCTAssertEqual(session.viewVisits.count, 2)
+        sendCIAppLog(session)
```

Comment:
It sends the log that is associated to CI App's test run:
<img width="1143" alt="Screenshot 2021-12-09 at 14 16 56" src="https://user-images.githubusercontent.com/2358722/145403447-d5e9a74a-d57e-441b-b5a4-c977e11a9b90.png">


Sample description looks this:
<img width="920" alt="Screenshot 2021-12-09 at 13 49 35" src="https://user-images.githubusercontent.com/2358722/145403317-44c219bf-e4ed-4fbf-8240-96ec38760e63.png">